docker/run-as-loxi

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+loxi_uid=$(stat -c'%u' /loxi/loxigen.py)
+loxi_gid=$(stat -c'%g' /loxi/loxigen.py)
+usermod -u $loxi_uid -g $loxi_gid loxi || echo "ok"
```

Comment:
I wasn't sure whether that would work even if the uid/gid are not assigned in /etc/passwd... Do you know?